doc/filters/replace.rst

```diff
@@ -1,12 +1,11 @@
 ``replace``
 ===========
 
-The ``replace`` filter formats a given string by replacing the placeholders
-(placeholders are free-form):
+The ``replace`` filter formats a given string by replacing the occurences of the search string with the replacement string:
 
 .. code-block:: jinja
 
-    {{ "I like %this% and %that%."|replace({'%this%': foo, '%that%': "bar"}) }}
+    {{ "I like this and that."|replace({'this': foo, 'that': "bar"}) }}
```

Comment:
Perhaps we can add a small paragraph recommending using some kind of conventions for placeholders to avoid wild replacements. Sometimes, the var where you want to do replacements comes from the DB or elsewhere.